### ISSUE TITLE
Buttons not working (JS)

### ISSUE THREAD

**Issue by brodycritchlow** [BatDev0/htmlwebsite#9]
recently added navigation buttons, and i cant get them to redirect.

```html

              <div class="container">
                <button class="btn btn1" id="Shop">Shop</button>
                <button class="btn btn1">Forums</button>
                <button class="btn btn1">Staff</button>
                <button class="btn btn1">Rules</button>
              </div>

```
```js
  document.getElementById("Store").onclick = function () {
    location.href = "tntsplash.net/store";
};

```

**Comment by brodycritchlow** [comment]
@dcritchlow

**Comment by dcritchlow** [comment]
You need either something like this
```<button onclick="myFunction()">Click me</button>```
With a function in your js 
Or the way you have it now you need to add the id “store” to one of the buttons and use window.location = tntsplash.net/store 
Location.href just sets where the button will go so for example if you have the button in a form then it would go to that location which is different than what you want. 

This is the way I would do it you could do is use a “a” link like this 
```<a href="https://tntsplash.net/store" class="button">Store</a>```

and style it to look like a button

a.button {
    -webkit-appearance: button;
    -moz-appearance: button;
    appearance: button;

    text-decoration: none;
    color: initial;
}

**Comment by brodycritchlow** [comment]
@dcritchlow 
Okay,
If i used the 
```html
<a href="https://store.tntsplash.net" class="button">Store</a>
```
Would i have to use JS as a redirecting guide?
What i mean by redirecting guide is,

Would i have to add some js to the `<a>` tag to redirect the page?

And would i have to add any other libraries in order to do

```css
a.button {
-webkit-appearance: button;
-moz-appearance: button;
appearance: button;

text-decoration: none;
color: initial;
}
``` 
or is it arleady in the main css.

**Comment by brodycritchlow** [comment]
@dcritchlow 
Also,
Would my animation on my button still work.

You can see the animation on our website:[tntsplash.net](url)

**Comment by brodycritchlow** [comment]
Resolved Issue #9
